bin/lighthouse/lighthouse-report.js

```diff
@@ -76,11 +76,20 @@ async function login(page) {
 
 async function setup(browser, context) {
   const page = await browser.newPage();
-  await page.setViewport({ width: 1440, height: 1000 });
-  await page.setCacheEnabled(true);
 
   try {
-    if (COUNTER !== 1) {
+    if (COUNTER === 0) {
+      // set a consent cookie for the cookie banner to not be displayed by default in mobile
+      await page.setCookie({
```

Comment:
I'm not sure we want this comment, as it is pretty clear what is going on. What do you think?